sgxtop.c

```diff
@@ -460,8 +461,9 @@ void enclaves_report(struct enclaves *enclaves)
 int main(int argc, char **argv)
 {
 	struct stats old, new;
-	struct enclaves *enclaves = enclaves_create(101);
+	struct enclaves *enclaves = NULL;
 
+	enclaves = enclaves_create(101);
```

Comment:
Can we create a Constant (MAX_ENCLAVES ?) for 101 ?

Reply:
Sure, good call.  This is actually the count of hash buckets.  It's kind of a big number for that purpose.